packages/drivers/local-driver/src/localDocumentDeltaConnection.ts

```diff
@@ -62,24 +62,31 @@ export class LocalDocumentDeltaConnection extends DocumentDeltaConnection {
           super(socket, documentId, new TelemetryNullLogger());
     }
 
+    protected submitCore(type: string, messages: IDocumentMessage[]) {
+        if (this.isBatchManagerDisabled) {
+            this.emitMessages(type, [messages]);
+        } else {
+            this.submitManager.add(type, messages);
+            this.submitManager.drain();
```

Comment:
It's not very clear why this dance of add + drain is required. Can we drop this implementation of submitCore in favor of parent's implementation?

Reply:
Not sure I follow. This is to keep the existing feature (batchmanager) as is, so I kept the original code here..

Reply:
Can this function be deleted, and thus parent submitCore() used? Will it work? Basically do we need this code?

Reply:
> Basically do we need this code?

I'm not sure and at this point I'm afraid to ask :))

Let me do some experimenting. I'd rather NOT do this, you have a point.

Reply:
There are some DDS tests failing consistently if these ops are not drained. Considering the code will be (eventually) removed and the `emit` above is basically `submitManager.drain` with fewer steps, I'll leave this as is.